examples/platform/alerts_common_scenarios.py

```diff
@@ -125,6 +125,7 @@ def main():
 
     # start by specifying Alert as the type of object to search
     alert_query = api.select(Alert)
+
```

Comment:
Around like 115, the permissions need to be updated to include
Alerts -ThreatMetadata - org.xdr.metadata